cosmos/testing/network/network.go

```diff
@@ -63,12 +63,20 @@ const (
 )
 
 var (
-	DummyContract   = common.HexToAddress("0x9fd0aA3B78277a1E717de9D3de434D4b812e5499")
-	TestKey, _      = ethsecp256k1.GenPrivKey()
-	ECDSATestKey, _ = TestKey.ToECDSA()
-	AddressFromKey  = TestKey.PubKey().Address()
-	Signer          = coretypes.LatestSignerForChainID(params.DefaultChainConfig.ChainID)
-	TestAddress     = crypto.PubkeyToAddress(ECDSATestKey.PublicKey)
+	DummyContract    = common.HexToAddress("0x9fd0aA3B78277a1E717de9D3de434D4b812e5499")
+	TestKey, _       = ethsecp256k1.GenPrivKey()
+	TestKey2, _      = ethsecp256k1.GenPrivKey()
+	TestKey3, _      = ethsecp256k1.GenPrivKey()
+	ECDSATestKey, _  = TestKey.ToECDSA()
+	ECDSATestKey2, _ = TestKey2.ToECDSA()
```

Comment:
refactor this into the fixture and make an array of addresses, as I mentioned before